app/src/main/java/org/p2p/wallet/history/ui/history/HistoryPresenter.kt

```diff
@@ -109,7 +111,7 @@ class HistoryPresenter(
     }
 
     private suspend fun fetchMoonpayTransactions() {
-        val moonpayItems = sellInteractor.loadUserSellTransactionsDetails()
+        val moonpayItems = sellTransactionsMapper.fromDomain(sellInteractor.loadUserSellTransactions())
```

Comment:
Why are you doing mapping in presenter?

Reply:
If you need items for adapter, maybe you'll move mapping to adapter?

Reply:
yeah, good idea
i think we don't have a strict rule where to do mapping for the adapter, i love the idea to do it in adapter